Services/UserService/Validators/DeleteUserRequestValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using DTO.RestRequests;
+using FluentValidation;
+
+namespace UserService.Validators
+{
+    public class DeleteUserRequestValidator: AbstractValidator<DeleteUserRequest>
+    {
+        public DeleteUserRequestValidator()
+        {
+            RuleFor(request => request.UserId)
+                .NotNull()
```

Comment:
NotEmpty ensures that the object isn't null.

Reply:
> Seems fine overall, have you tested it?
Yeah, it's okay